hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -219,81 +208,6 @@ public void testInitializeCatalogWithProperties() {
         .isEqualTo("/user/hive/testwarehouse");
   }
 
-  @Test
-  public void testCreateTableTxnBuilder() throws Exception {
```

Comment:
why are these all being removed? They weren't removed in the earlier version of the PR you had

Reply:
Few of the tests are being referred within the CatalogTests with different name. 
like 
testCreateTableTxnBuilder -> [testCreateTransaction](https://github.com/apache/iceberg/blob/7240752e18278b3454be141c2e4b121079004d8b/core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java#L1486)